modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/milker.dm

```diff
@@ -0,0 +1,300 @@
+			pain_amount = PAIN_LIGHT
+			squirt_volume = VOLUME_LIGHT
+		if(HANDLE_STRONG)
+			if(user)
+				user.visible_message(
+				message = span_purple("[user] tightens [user.p_their()] jaw and clenches [user.p_their()] milking hand, appearing to opt for a more aggressive approach."),
+				self_message = span_purple("You tighten your jaw and clench your milking hand, opting for a more aggressive approach."),
+				blind_message = span_purple("You hear a vigorous rustle."),
+				vision_distance = 1)
+			desc = "[initial(desc)]" + "\n[span_purple("It looks rather aggressive!")]"
+			pleasure_amount = PLEASURE_STRONG
+			arousal_amount = AROUSAL_STRONG
+			pain_amount = PAIN_STRONG
+			squirt_volume = VOLUME_STRONG
+
+/// Checks if the breasts are present, exposed, lactating, in range, and containing some kind of fluid. Returns a message if not!
+/obj/item/milker/proc/breast_check(mob/living/carbon/being_milked, mob/living/carbon/human/milker, obj/item/organ/genital/breasts/breasts)
+	var/self_suckle = (being_milked == milker) // Grammarize
+	if(!being_milked.client)
+		to_chat(milker, span_warning("[being_milked] is too busy staring off into space to be milked!"))
+		return FALSE
+	if(!being_milked.client.prefs.read_preference(/datum/preference/toggle/master_erp_preferences))
+		to_chat(milker, span_warning("[self_suckle ? "You would prefer to leave those alone!" : "[being_milked] would prefer you leave those alone!"]"))
+		return FALSE
+	if(!being_milked.client.prefs.read_preference(/datum/preference/toggle/erp/sex_toy)) // I guess its a sextoy!
+		to_chat(milker, span_warning("[self_suckle ? "You would prefer to put your hands somewhere else!" : "[being_milked] would prefer you to keep your hands to yourself!"]"))
+		return FALSE
+	if(!ishuman(being_milked)) // Trying to milk a robot? A megarachnid? *IAN*?
+		to_chat(milker, span_warning("You can't milk that!"))
+		return FALSE
+	if(being_milked.stat != CONSCIOUS)
+		to_chat(milker, span_warning("[self_suckle ? "You are in no condition to be milked!" : "[being_milked] is in no condition to be milked!"]"))
+		return FALSE
+	if(!istype(breasts))
+		to_chat(milker, span_warning("[self_suckle ? "You don't have any breasts!" : "[being_milked] doesn't seem to have any breasts!"]"))
+		return FALSE
+	if(!breasts.is_exposed())
+		to_chat(milker, span_warning("[self_suckle ? "You can't get to your breasts!" : "[being_milked]'s breasts aren't accessible!"]"))
+		return FALSE
+	if(!breasts.lactates)
+		to_chat(milker, span_warning("[self_suckle ? "You aren't lactating" : "[being_milked] doesn't seem to be lactating!"]"))
+		return FALSE
+	if(breasts.internal_fluids.total_volume <= 0)
+		to_chat(milker, span_warning("[self_suckle ? "You're out of milk!" : "[being_milked] is fresh out of milk!"]"))
+		return FALSE
+	if(!in_range(being_milked, milker))
+		to_chat(milker, span_warning("[self_suckle ? "Your breasts are too far away! ...somehow!" : "[being_milked]'s breasts are too far away!"]"))
+		return FALSE
```

Comment:
I disagree, I feel like more feedback as to why something isnt working is a good thing. As someone who regularly handles erp-related ahelps, I feel it'd help solve those before they happen.

Also "You can't do that right now!" could imply it could be doable later, which isn't true if they dont have any breasts.

Reply:
Fair.